### PR TITLE
Update FlaskAppBuilder to v3

### PR DESCRIPTION
**Pull request by ephraimbuddy**
---
This PR updates flask_appbuilder to version 3.0.
The stable API in development have been updated to use the latest marshmallow(3.6.1). 
This is the only breaking change I can find.

Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Target Github ISSUE in description if exists
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

**Comment by ephraimbuddy** [comment]
@mik-laj , please can you take a look?

File: requirements/requirements-python3.6.txt [hunk 1]
```diff
@@ -57,11 +57,11 @@ azure-common==1.1.25
 azure-cosmos==3.1.2
 azure-datalake-store==0.0.48
 azure-kusto-data==0.0.45
-azure-mgmt-containerinstance==1.5.0
+azure-mgmt-containerinstance==2.0.0
```
Comment (mik-laj) [inline requirements/requirements-python3.6.txt]: This library breaks build.

File: requirements/requirements-python3.6.txt [hunk 2]
```diff
@@ -100,13 +100,13 @@ croniter==0.3.34
...
-dask==2.19.0
-datadog==0.37.1
+dask==2.20.0
+datadog==0.38.0
 decorator==4.4.2
 defusedxml==0.6.0
 dill==0.3.2
 distlib==0.3.1
-distributed==2.19.0
+distributed==2.20.0
```
Comment (ephraimbuddy) [inline requirements/requirements-python3.6.txt]:
```suggestion
distributed==2.19.0
```

File: requirements/requirements-python3.7.txt [hunk 3]
```diff
@@ -99,13 +99,13 @@ croniter==0.3.34
...
-dask==2.19.0
-datadog==0.37.1
+dask==2.20.0
+datadog==0.38.0
 decorator==4.4.2
 defusedxml==0.6.0
 dill==0.3.2
 distlib==0.3.1
-distributed==2.19.0
+distributed==2.20.0
```
Comment (ephraimbuddy) [inline requirements/requirements-python3.7.txt]:
```suggestion
distributed==2.19.0
```

File: requirements/requirements-python3.8.txt [hunk 4]
```diff
@@ -99,13 +99,13 @@ croniter==0.3.34
...
-dask==2.19.0
-datadog==0.37.1
+dask==2.20.0
+datadog==0.38.0
 decorator==4.4.2
 defusedxml==0.6.0
 dill==0.3.2
 distlib==0.3.1
-distributed==2.19.0
+distributed==2.20.0
```
Comment (ephraimbuddy) [inline requirements/requirements-python3.8.txt]:
```suggestion
distributed==2.19.0
```

File: requirements/requirements-python3.6.txt [hunk 5]
```diff
@@ -90,7 +90,7 @@ chardet==3.0.4
 click==6.7
 clickclick==1.2.2
 cloudant==2.13.0
-cloudpickle==1.4.1
+cloudpickle==1.5.0
```
Comment (ephraimbuddy) [inline requirements/requirements-python3.6.txt]:
```suggestion
cloudpickle==1.4.1
```

File: requirements/requirements-python3.7.txt [hunk 6]
```diff
@@ -90,7 +90,7 @@ chardet==3.0.4
 click==6.7
 clickclick==1.2.2
 cloudant==2.13.0
-cloudpickle==1.4.1
+cloudpickle==1.5.0
```
Comment (ephraimbuddy) [inline requirements/requirements-python3.7.txt]:
```suggestion
cloudpickle==1.4.1
```

File: requirements/requirements-python3.8.txt [hunk 7]
```diff
@@ -90,7 +90,7 @@ chardet==3.0.4
 click==6.7
 clickclick==1.2.2
 cloudant==2.13.0
-cloudpickle==1.4.1
+cloudpickle==1.5.0
```
Comment (ephraimbuddy) [inline requirements/requirements-python3.8.txt]:
```suggestion
cloudpickle==1.4.1
```

File: setup.py [hunk 8]
```diff
@@ -215,7 +215,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'cloudant>=2.0',
 ]
 dask = [
-    'distributed>=2.11.1, <3',
+    'distributed>=2.11.1, <2.20',
 ]
```
Comment (ephraimbuddy) [inline setup.py]:
```suggestion
    'distributed>=2.11.1, <2.20',
    'cloudpickle>=1.4.1, <1.5.0'
]
```

File: setup.py [hunk 9]
```diff
@@ -216,6 +216,7 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
...
 dask = [
     'distributed>=2.11.1, <2.20',
+    'cloudpickle>=1.4.1, <1.5.0'
```
Comment (ephraimbuddy) [inline setup.py]:
```suggestion
    'cloudpickle>=1.4.1, <1.5.0',
    'distributed>=2.11.1, <2.20'
```

File: setup.py [hunk 10]
```diff
@@ -215,7 +215,8 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
     'cloudant>=2.0',
 ]
 dask = [
-    'distributed>=2.11.1, <3',
+    'distributed>=2.11.1, <2.20',
+    'cloudpickle>=1.4.1, <1.5.0'
```
Comment (ephraimbuddy) [inline setup.py]:
```suggestion
    'cloudpickle>=1.4.1, <1.5.0',
    'distributed>=2.11.1, <2.20'
```

**Comment by mik-laj** [comment]
It took me some time to check if the Javascript code was working correctly, but everything looks very good. Nice change!

**Review by mik-laj — APPROVED** [review]
It took me some time to check if the Javascript code was working correctly, but everything looks very good. Nice change!

**Comment by potiuk** [comment]
Fantastic!

**Pull request merged**